features/step_definitions/basic_steps.rb

```diff
@@ -1,3 +1,13 @@
+  Given ("I visit the page") do
+    visit root_path
+  end
+
+  Given("following users exist") do |table|
```

Comment:
When are you using this step

Reply:
it's used in user_must_login_to_use_the_app.feature